### ISSUE TITLE
[Feature Request] "/sub anime"-command via emoji under episode release messages

### ISSUE THREAD

**Issue by D1yt** [r-anime/ZeroTsu#69]
Hey, instead of the /sub command no one will ever find, could someone add an emoji/button you could react to/press posted under new episode messages which would act as a toggle (just like normally)?

**Comment by apiks** [comment]
While not a terrible idea there is a few problems that make this unlikely to happen:

- The bot is no longer in active development
- People can't know about what the reactions mean magically and will probably click them randomly and be surprised
- The new episodes messages are logically separate from the sub command
- People are supposed to find the /sub command by doing the /help command first. Slash commands has made this more difficult and uglier, but that is the proper way of doing it
- A possible solution is to create the ability for admins/mods to toggle whether new episode messages to do this react thing

In short while it sounds nice it is slightly pointless and will bring more problems than the small solution it gives. Unfortunately even if I agreed to it I don't think I'll be implementing new features anytime soon.
